requirements.txt

```diff
@@ -27,6 +28,7 @@ packaging>=21.0
 pillow==8.4.0
 psycogreen==1.0.2
 psycopg2-binary==2.9.2
+qrcode==7.3.1
```

Comment:
I think we can merge this with the `django-otp` by using `django-otp[qrcode]=1.1.3`

Reply:
Good proposal